hevm.cabal

```diff
@@ -263,5 +263,55 @@ common test-base
     array,
     vector,
     smt2-parser >= 0.1.0.1,
-    stm,
-    spawn
+    stm >= 2.5.0,
+    spawn >= 0.3,
+    witherable,
+    smt2-parser >= 0.1.0.1
+
+library test-utils
```

Comment:
I don't understand why this is showing up in the diff?

Reply:
Sorry, was a merge mistake :( Thanks for catching it!

Reply:
Fixed :)